src/apps/backend/modules/task/task-service.ts

```diff
@@ -18,12 +18,12 @@ export default class TaskService {
     return TaskWriter.deleteTask(params);
   }
 
-  public static async getTaskForAccount(params: GetTaskParams): Promise<Task> {
-    return TaskReader.getTaskForAccount(params);
+  public static async updateTask(params: UpdateTaskParams): Promise<Task> {
+    return TaskWriter.updateTask(params);
   }
 
-  public static async getTaskByNameForAccount(params: GetTaskByNameParams): Promise<Task> {
-    return TaskReader.getTaskByNameForAccount(params);
+  public static async getTaskForAccount(params: GetTaskParams): Promise<Task> {
```

Comment:
There is potentially wrong with the function name.

An account can have multiple task, right?

Reply:
In the case of a single account, we are able to add more than one task as the title and description.